### ISSUE TITLE
Cargo wagon item limit

### ISSUE THREAD

**Issue by jxu** [deniszholob/factorio-cheat-sheet#418]
https://factoriocheatsheet.com/#cargo-wagon-transfer

I couldn't find the number. Maybe I missed it. But what to remember is 2000 ore per wagon (or 4000 plates per wagon)
50 ore per slot x 40 slots. I always forget how big a steel chest / logistic chest is too. It's 48 slots
A good default setup is 1 locomotive and 4 cargo wagons.  

Very useful guide: [Beginners Guide to Rail by piper.spirit](https://steamcommunity.com/sharedfiles/filedetails/?id=2737259470)

**Comment by deniszholob** [comment]
Also some1 else mentioned to include this to which i agree, so writing down to remember to include:
From that guide:
"Chain Signal Into Junction, Rail Signal Out."

**Comment by jxu** [comment]
yes absolutely. (the reason why it works is because you never want a train blocking an intersection, so the chain signal reads the rail signal at the exit to ensure the junction is free)

**Comment by HonkingGoose** [comment]
> Also some1 else mentioned to include this to which i agree, so writing down to remember to include:
From that guide:
"Chain Signal Into Junction, Rail Signal Out."

See:

- https://github.com/deniszholob/factorio-cheat-sheet/issues/401
